go/private/go_toolchain.bzl

```diff
@@ -200,6 +200,33 @@ def declare_bazel_toolchains(
         visibility = ["//visibility:private"],
     )
 
+    sdk_name_label = Label("//go/toolchain:sdk_name")
+
+    native.config_setting(
+        name = prefix + "match_sdk_name",
+        flag_values = {
+            sdk_name_label: go_toolchain_repo[1:] if go_toolchain_repo.startswith("@") else go_toolchain_repo,
```

Comment:
I would like to avoid this kind of ad-hoc parsing. Can we arrange for the correct name to be passed in as an argument to the function?